backend/tournesol/serializers.py

```diff
@@ -38,8 +40,15 @@ class Meta:
             "language",
             "rating_n_ratings",
             "rating_n_contributors",
+            "duration",
         ]
 
+    # Convert duration to seconds to facilitate use of Humanize package
+    def get_duration(self, obj):
+        if obj.duration:
+            return int(obj.duration.total_seconds())
+        return 0
```

Comment:
A type hint is necessary, so as to the correct type appears in the OpenAPI schema.  
Also, I would recommend to return `null` when the duration is not defined, to make sure the frontend handles this case explicitly.
```suggestion
    # Convert duration to seconds to facilitate use of Humanize package
    def get_duration(self, obj) -> Optional[int]:
        if obj.duration:
            return int(obj.duration.total_seconds())
        return None
```